### PR TITLE
Update s3 config

### PR DESCRIPTION
**Pull request by lonnylot**
Updated S3 config to be in line with https://github.com/NativePHP/electron/pull/65

### CODE REVIEW

File: config/nativephp.php [hunk 1]
```diff
@@ -58,12 +58,12 @@
         'providers' => [
             's3' => [
                 'driver' => 's3',
-                'key' => env('AWS_ACCESS_KEY_ID'),
-                'secret' => env('AWS_SECRET_ACCESS_KEY'),
+                'profile' => env('AWS_PROFILE'),
```
Comment (shanerbaner82) [inline config/nativephp.php:61]: Why are you doing this instead of the key/secret?

Comment (lonnylot) [reply config/nativephp.php:61]: Because I didn't know if the key / secret were compiled in the app so I opted for the profile

**Comment by simonhamp** [comment]
But won't the AWS profile get compiled into the app?

How is it different/better?

**Comment by lonnylot** [comment]
> But won't the AWS profile get compiled into the app?
> 
> How is it different/better?

AH ok. I understand the question now.

The profile is  just a key that maps to credentials on my local computer. So it has no value outside of my personal machine.

IIRC after I submitted these PRs there was a way to exclude env vars from the compiled version? Is that accurate? If so I can take a look later this week and update both PRs to exclude these values. The problem w/ that is that there are cases where you _may_ want to share keys (e.g. if you want to allow people to read from a private S3, or upload to s3, or ¯\_(ツ)_/¯ )

**Comment by simonhamp** [comment]
Yeh, as you suggested, I think the better approach will be to make sure these keys aren't distributed with the app. They really only exist here for the developer to deploy their compiled application to S3.

For the user to download from S3 when running the app, they don't (or at least _shouldn't_) need these keys at all. If you do need to share keys, I think we'll need to find a way to use the user's local secure storage (e.g. Keychain on Mac) and you provide a way in-app to allow the user to paste in the codes securely.

**Comment by lonnylot** [comment]
> If you do need to share keys

Depending on the app there are definitely situations where you would share. Eg. an internal app for managing something

**Comment by simonhamp** [comment]
@lonnylot Sure, and in that case it should be baked into your app's own UI and business logic, not a part of the NativePHP framework

**Pull request closed without merging**